README.md

```diff
@@ -22,7 +24,7 @@ $ npm install --save sal.js
 $ yarn add sal.js
 ```
 
-Then load it with your favorite module loader or use as global, when no modules
+Load it with your favorite module loader or use as a global
```

Comment:
Maybe `as a global variable`?

Reply:
At first I actually thought of explicitely documenting the "as a global" use-case. It could look like this (along with importing styles):

---

Load it with your favorite module loader:

```js
// ES6 modules
import sal from 'sal.js'

// CommonJS modules
var sal = require('sal.js')
```

or use as a global variable:

```html
<script src="sal.js"></script>
```

Import the styles in your preprocessor:

```scss
// Webpack
@import '~sal.js/sal.css';

// Other
@import './node_modules/sal.js/dist/sal.css';
```

or in your document's `head`:

```html
<link rel="stylesheet" type="text/css" href="sal.css" />
```

What do you think?

Reply:
I would like to emphasise on using `sal.js` with Webpack, rather than as a global script. I'm planning to make a Wiki docs for that kind of tips and tricks. For now you can skip it.